.gitignore

```diff
@@ -1,2 +1,3 @@
 bin/
 obj/
+*.png
```

Comment:
Are you sure you want to ignore *all* `.png` files? This could be annoying in the future if you want e.g. PNG icons. I think a good practice for hiding output data from source control is to quarantine it in an `out` folder and gitignoring that one.